gemini/button-test.js

```diff
@@ -0,0 +1,68 @@
+/**
+Copyright 2016 Autodesk,Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+*/
+
+gemini.suite('default button', (suite) => {
+  suite
+    .setUrl('/iframe.html?selectedKind=Button&selectedStory=with%20text')
+    .setCaptureElements(".example")
+    .before(function(actions, find) {
+      this.button = find('orion-button');
+    })
+    .capture('default')
+    .capture('hovered', function(actions, find){
```

Comment:
This is more for gemini rather than storybook. The string passed into capture becomes the name of the directory that gemini saves the screenshot to.